randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -673,7 +681,9 @@ Extra - room_id: [17]
   > Pickup (Missile Tank)
       Can Use Power Bombs
   > Door to Evir Shaft
-      Combat (Intermediate) or Can Kill Tough Beam-Resistant Enemy
+      Any of the following:
```

Comment:
you forgot to add these in the other connection too (evir shaft -> aquaruium shaft west)